### DISCUSSION TITLE
内置节点text-block怎么动态更改内部text的颜色字体大小，以及文字内容

### DISCUSSION THREAD

**Discussion by Epic-Deno** [antvis/X6#3410] [Q&A] [answered]
目前我通过更改这个文字节点内的属性， 但是替换完属性之后不能立刻更新视图，需要拖动或者改变节点的大小才可以变化内部视图，有没有API可以直接更新节点的内容
![image](https://user-images.githubusercontent.com/74635349/226509088-4f7cc3b5-b98a-4398-8969-87ec80700789.png)

**Comment by Epic-Deno** [comment, marked as answer]
下午我想了一办法就是通过updateCellId,先拷贝之前节点的属性，然后通过这个API去更新，基本实现我的预期效果，还有没有其他的方法
![image](https://user-images.githubusercontent.com/74635349/226538757-53ae0f2b-e1cc-4166-8d78-9b2c4a035bbd.png)

**Reply by NewByVector** [reply]
这样就可以。
```ts
node.attr('label/text', 'xxx')
```
